docs/elements.md

```diff
@@ -1161,6 +1161,7 @@ def square(x):
 | `highlight-lines-color` | text    | `"#b3d7ff"`     | Specifies the color of highlighted lines of code.                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `copy-code-button`      | boolean | false           | Whether to include a button to copy the code displayed by this element.                                                                                                                                                                                                                                                                                                                                                                                   |
 | `show-line-numbers`     | boolean | false           | Whether to show line numbers in code displayed by this element.                                                                                                                                                                                                                                                                                                                                                                                           |
+| `normalize-whitespace`  | boolean | false           | Whether to strip leading/trailing whitespace and dedent the contents. Recommended in most situations.                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
I think this should be recommended for cases where the code is inline in the question file. For cases with `source-file-name` it may not be needed.

Reply:
Discussion added here: https://github.com/PrairieLearn/PrairieLearn/pull/11418/commits/696e27580d3c44517ec894b88f96eae936730cd7#diff-ef20c77a77d5b9b229d9f077842a5215a646f9b93fa89a36a3bc9144972188ce